test/server/utils/routes.test.js

```diff
@@ -64,6 +64,10 @@ describe('routes util', () => {
     });
   });
 
+  it('GET request to invalidate endpoint', (done) => {
+    req.get('/invalidate').expect(200, done);
+  });
```

Comment:
Will be great to test what we really do invalidate - get stats before do request and do get stats again

Reply:
I can't detect any changes in `server.middleware`

Reply:
and `server.middleware.context.stats` is always `undefined`

Reply:
Strange, should be no

Reply:
I've thought about an Idea maybe we can create an option `invalidateCallback` and we create a function that creates a file then delete after confirming that the file exists.

Reply:
Bad idea, maybe we need fix something in tests, need debug